README.md

```
@@ -2,133 +2,125 @@
 
-The preprocessing bottleneck has been completely eliminated through the mel worker architecture. See [docs/mel-worker-architecture.md](docs/mel-worker-architecture.md) for details.
+Preprocessing is fully offloaded via the mel worker. See [docs/mel-worker-architecture.md](docs/mel-worker-architecture.md) for details.
 
 ---
 
 ## Architecture
 
-### Multi-Worker Pipeline
+### Multi-worker pipeline
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)